app/models/form/account_batch.rb

```diff
@@ -25,19 +26,21 @@ def save
       suppress_follow_recommendation!
     when 'unsuppress_follow_recommendation'
       unsuppress_follow_recommendation!
+    when 'suspend'
+      suspend!
```

Comment:
I know this is not a new thing but I'm slightly uncomfortable with having moderation actions and individual follower/following management in the same class.